symengine/cwrapper.h

```diff
@@ -256,9 +256,26 @@ void basic_subs2(basic s, const basic e, const basic a, const basic b);
 //! The caller is responsible to free the pointer using 'basic_str_free'.
 char* ascii_art_str();
 
+//! Greatest Common Divisor
+void ntheory_gcd(basic s, const basic a, const basic b);
+//! Least Common Multiple
+void ntheory_lcm(basic s, const basic a, const basic b);
+//! \return next prime after `a`
+void ntheory_nextprime(basic s, const basic a);
+//! modulo round toward zero
+void ntheory_mod(basic s, const basic n, const basic d);
+//! \return quotient round toward zero when `n` is divided by `d`
+void ntheory_quotient(basic s, const basic n, const basic d);
+//! Fibonacci number
+void ntheory_fibonacci(basic s, unsigned long a);
```

Comment:
Add a clarification in the documentation where the sequence starts from, 0 or 1, and also if we are counting with 0 index as first element or 1 index as the first.